### ISSUE TITLE
Icons in Flex Toolbar broken

### ISSUE THREAD

**Issue by onetrev** [ignism/nucleus-dark-ui#17]
When I activate Nucleus Dark UI, some of my [Flex Toolbar](https://github.com/cakecatz/flex-toolbar) icons no longer display.

![untitled-0](https://cloud.githubusercontent.com/assets/597601/9695651/b9e2fe9e-5318-11e5-855f-ab27d5d906e9.png)

**Comment by ignism** [comment]
Hey, in order to reproduce this, mind if I ask what icons you were using in the config file? Or better yet, share me the config file so I can recreate this exact toolbar?

**Comment by onetrev** [comment]
Thank you for looking into to this! Not sure how you wanted me to share the config file to you, so I'm just going ahead and pasting this beast in here...

``` [
  {
    "type": "button",
    "icon": "embed",
    "callback": "emmet:wrap-with-abbreviation",
    "tooltip": "Emmet Wrap",
    "iconset": "icomoon"
  },
  {
    "type": "button",
    "icon": "embed2",
    "callback": "emmet:interactive-expand-abbreviation",
    "tooltip": "Interactive Emmet Wrap",
    "iconset": "icomoon"
  },
  {
    "type": "button",
    "icon": "indent-more",
    "callback": "editor:auto-indent",
    "tooltip": "Auto Indent",
    "iconset": "fi"
  },
  {
    "type": "button",
    "icon": "map",
    "callback": "symbols-view:toggle-file-symbols",
    "tooltip": "File Symbols",
    "iconset":"ion"
  },
  {
    "type": "button",
    "icon": "android-color-palette",
    "callback": "pigments:show-palette",
    "tooltip": "Pigments",
    "iconset":"ion"
  },
  {
    "type": "button",
    "icon": "bookmark",
    "callback": "bookmarks:toggle-bookmark",
    "tooltip": "Toggle Bookmark",
    "iconset":"ion"
  },
  {
    "type": "button",
    "icon": "check-square-o",
    "callback": "imdone-atom:tasks",
    "tooltip": "Project To Do List",
    "iconset":"fa"
  },
  {
    "type": "button",
    "icon": "circuit-board",
    "callback": "git-diff:toggle-diff-list",
    "tooltip": "Git Diffs",
  },
  {
    "type": "spacer"
  },
  {
    "type": "button",
    "icon": "three-bars",
    "tooltip": "Toggle Command Palette"
    "callback": ["command-palette:toggle"]
  },
  {
    "type": "button",
    "icon": "file-submodule",
    "tooltip": "Toggle Projects"
    "callback": ["project-manager:list-projects"]
  },
  {
    "type": "url",
    "icon": "wordpress-plain",
    "url": "https://codex.wordpress.org/Function_Reference",
    "iconset": "devicon"
  },
  {
    "type": "button",
    "icon": "search",
    "iconset": "ion",
    "tooltip": "Find in Project"
    "callback": ["project-find:show"]
  },
  {
    "type": "button",
    "icon": "insert-template",
    "iconset": "icomoon",
    "tooltip": "Split Down"
    "callback": ["pane:split-down"]
  },
  {
    "type": "button",
    "icon": "terminal",
    "tooltip": "Toggle Dev Tools"
    "callback": ["window:toggle-dev-tools"]
  },
  {
    "type": "button",
    "icon": "markdown",
    "callback": "markdown-preview:toggle",
    "hide": "!markdown"
  },
  {
    "type": "button",
    "icon": "document",
    "callback": "application:new-file",
    "tooltip": "New File",
    "iconset": "ion",
    "mode": "dev"
  },
  {
    "type": "button",
    "icon": "refresh",
    "callback": "window:reload",
    "tooltip": "Reload Window",
    "iconset": "ion",
    "mode": "dev"
  },
  {
    "type": "spacer"
  },
  {
    "type": "button",
    "icon": "gear-a",
    "callback": ["settings-view:open"],
    "iconset": "ion"
  }
]
```

**Comment by ignism** [comment]
Perfect! thank you, I have the same issue and I am looking into this now.

**Comment by ignism** [comment]
Patched in version 0.2.13

**Comment by onetrev** [comment]
Awesome, thanks for all the fixes! Now you just need to get your donation link fixed. :)
